test/common/grpc/grpc_client_integration.h

```diff
@@ -125,6 +127,27 @@ class DeltaSotwIntegrationParamTest
   SotwOrDelta sotwOrDelta() const { return std::get<2>(GetParam()); }
 };
 
+class EdsModeDeltaSotwIntegrationParamTest
```

Comment:
Consider renaming to `MultiplexedDeltaSotwIntegrationParamTest` as this is not only for EDS.